task/04-arrays-tasks.js

```diff
@@ -559,7 +590,10 @@ function getElementByIndexes(arr, indexes) {
  * 
  */
 function swapHeadAndTail(arr) {
-    throw new Error('Not implemented');
+   let middle = Math.floor(arr.length/2);
+   let head = arr.splice(0,middle);
```

Comment:
if you do it with a splice the function has a side-effect - it changes the original array, which is a bad practice. If you use splice you need to copy array to avoid that. Or you could implemtn this using slice